questionnaire/questionnaire-service.js

```diff
@@ -47,6 +49,25 @@ function createQuestionnaireService({
         const uuidV4 = uuidv4();
         const questionnaire = templates[templateName](uuidV4);
 
+        if (apiVersion === '2023-05-17') {
+            if (!owner) {
+                throw new VError(
+                    {
+                        name: 'OwnerNotFound'
+                    },
+                    `Owner data must be defined`
+                );
+            }
+
+            questionnaire.answers = {
+                ...questionnaire.answers,
+                owner: {
+                    'owner-id': owner.id,
```

Comment:
I know we had a chat around the hyphen stuff... is it worth reverting to `ownerId` when storing it on the questionnaire?